2017-2018/GameEngine/graphics/visualizer.py

```diff
@@ -20,37 +29,37 @@ def __init__(self, print_walls, print_pacman, loop=None):
+                'l': SpriteStripAnim('graphics/pac_t.bmp', (487,112,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'u': SpriteStripAnim('graphics/pac_t.bmp', (520,112,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'd': SpriteStripAnim('graphics/pac_t.bmp', (552,112,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY)))
             },
             'pink': {
-                'r': SpriteStripAnim('graphics/pac.bmp', (456,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
-                'l': SpriteStripAnim('graphics/pac.bmp', (487,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
-                'u': SpriteStripAnim('graphics/pac.bmp', (520,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
-                'd': SpriteStripAnim('graphics/pac.bmp', (552,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY)))
+                'r': SpriteStripAnim('graphics/pac_t.bmp', (456,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'l': SpriteStripAnim('graphics/pac_t.bmp', (487,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'u': SpriteStripAnim('graphics/pac_t.bmp', (520,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'd': SpriteStripAnim('graphics/pac_t.bmp', (552,80,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY)))
             },
             'blue': {
-                'r': SpriteStripAnim('graphics/pac.bmp', (456,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
-                'l': SpriteStripAnim('graphics/pac.bmp', (487,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
-                'u': SpriteStripAnim('graphics/pac.bmp', (520,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
-                'd': SpriteStripAnim('graphics/pac.bmp', (552,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY)))
+                'r': SpriteStripAnim('graphics/pac_t.bmp', (456,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'l': SpriteStripAnim('graphics/pac_t.bmp', (487,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'u': SpriteStripAnim('graphics/pac_t.bmp', (520,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY))),
+                'd': SpriteStripAnim('graphics/pac_t.bmp', (552,96,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY)))
             },
             'frightened': {
-                'r': SpriteStripAnim('graphics/pac.bmp', (584,64,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY)))
+                'r': SpriteStripAnim('graphics/pac_t.bmp', (584,64,16,16), 2, 1, True, int(1/(SPRITE_FREQUENCY * DISPLAY_FREQUENCY)))
```

Comment:
Perhaps this filename should be a constant variable, instead of being hardcoded all over here...

Reply:
:+1: